src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  let notMarriage = `I am not married`;
+
+  if (isMarried) {
+    notMarriage = `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`;
+  }
```

Comment:
You can move it to a function that returns your needed text. using let in global scope isn't a good idea.
Or you can use ternary inside JSX ( worse variant)